test/functional/ui/float_spec.lua

```diff
@@ -10160,6 +10160,72 @@ describe('float window', function()
       command('fclose!')
       -- it is currently not supported.
       eq('Vim(set):E474: Invalid argument: winborder=custom', pcall_err(command, 'set winborder=custom'))
+
+      command('set winborder=bold')
```

Comment:
Do we really need 100 lines of test code for this? Now I'm wondering how much other test code exists for the other variations. 

We only need 1 or 2 tests that test the variations. And then the rest of the variations can be *exercised* with 1 line of test code, and maybe 1 or 2 assertions, but they don't need 100 lines of screen test code.

Reply:
I have cleaned up some

Reply:
No they're not